src/Runner/TaskResultInterface.php

```diff
@@ -17,10 +17,7 @@ public function isPassed(): bool;
 
     public function isBlocking(): bool;
 
-    /**
-     * @return null|string
-     */
-    public function getMessage();
+    public function getMessage(): string;
```

Comment:
The docblock says null or string, shouldn't it be `: ?string` ?

Reply:
It's now `string` in the implementing `TaskResult` class

Reply:
Ok, if the users aren't using this interface, this should be fine.